### ISSUE TITLE
Following instructions Exactly (with Rails API)

### ISSUE THREAD

**Issue by jancel** [tpitale/swagger_yard-rails#8]
Hey guys.  I would like to use this gem with Rails (generated with the --api version).  After following the steps as described (with the Mount in my Routes), I'm getting resource not found.

```ruby
#Gemfile
gem 'swagger_yard-rails'
```

```ruby
# config/initializers/swagger_yard.rb
SwaggerYard.configure do |config|
  config.api_version = "1.0"
  config.reload = Rails.env.development?

  # where your actual api is hosted from
  config.api_base_path = "http://localhost:5000/api/v1"
end
```

```ruby
#config/routes.rb
mount SwaggerYard::Rails::Engine, at: "/swagger"
```

```bash
$ rails routes
      swagger_yard_rails          /swagger                               SwaggerYard::Rails::Engine
```

When I run the engine, I get path/resource not found

```bash
$ rails s
=> Booting Puma
=> Rails 5.0.1 application starting in development on http://localhost:3000
=> Run `rails server -h` for more startup options
W, [2017-02-13T13:20:36.832284 #72384]  WARN -- : You are setting a key that conflicts with a built-in method OmniAuth::AuthHash::InfoHash#name defined at /Users/jancel/.rvm/gems/ruby-2.4.0@protoss/gems/omniauth-1.3.2/lib/omniauth/auth_hash.rb:34. This can cause unexpected behavior when accessing the key via as a property. You can still access the key via the #[] method.
[72384] Puma starting in cluster mode...
[72384] * Version 3.7.0 (ruby 2.4.0-p0), codename: Snowy Sagebrush
[72384] * Min threads: 5, max threads: 5
[72384] * Environment: development
[72384] * Process workers: 2
[72384] * Preloading application
[72384] * Listening on tcp://0.0.0.0:5000
[72384] Use Ctrl-C to stop
[72384] - Worker 0 (pid: 72417) booted, phase: 0
[72384] - Worker 1 (pid: 72418) booted, phase: 0
Started GET "/swagger" for 127.0.0.1 at 2017-02-13 13:20:43 -0600
  ActiveRecord::SchemaMigration Load (0.4ms)  SELECT "schema_migrations".* FROM "schema_migrations"

ActionController::RoutingError (No route matches [GET] "/swagger"):

/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/actionpack/lib/action_dispatch/middleware/debug_exceptions.rb:53:in `call'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/actionpack/lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/railties/lib/rails/rack/logger.rb:36:in `call_app'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/railties/lib/rails/rack/logger.rb:24:in `block in call'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/activesupport/lib/active_support/tagged_logging.rb:69:in `block in tagged'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/activesupport/lib/active_support/tagged_logging.rb:26:in `tagged'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/activesupport/lib/active_support/tagged_logging.rb:69:in `tagged'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/railties/lib/rails/rack/logger.rb:24:in `call'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/actionpack/lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.1) lib/rack/runtime.rb:22:in `call'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/activesupport/lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/actionpack/lib/action_dispatch/middleware/executor.rb:12:in `call'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/actionpack/lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.1) lib/rack/sendfile.rb:111:in `call'
/Users/jancel/.rvm/gems/ruby-2.4.0@protoss/bundler/gems/rails-24c585df91a5/railties/lib/rails/engine.rb:522:in `call'
puma (3.7.0) lib/puma/configuration.rb:226:in `call'
puma (3.7.0) lib/puma/server.rb:578:in `handle_request'
puma (3.7.0) lib/puma/server.rb:415:in `process_client'
puma (3.7.0) lib/puma/server.rb:275:in `block in run'
puma (3.7.0) lib/puma/thread_pool.rb:120:in `block in spawn_thread'

```

**Comment by jancel** [comment]
After updating to edge (here), I was able to see a few more routes which are nested.  Visiting /swagger/swagger worked as defined.

**Comment by nicksieger** [comment]
It looks like current master is released as 0.3.3. So are you saying you resolved the issue? Please close this if so. If not, please provide more details.

**Comment by jancel** [comment]
Sure is.
